bazel/repository_locations.bzl

```diff
@@ -1177,31 +1177,99 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         use_category = ["dataplane_ext"],
         extensions = ["envoy.wasm.runtime.v8"],
-        release_date = "2022-10-12",
+        release_date = "2024-05-31",
+        cpe = "N/A",
+    ),
+    dragonbox = dict(
+        project_name = "Dragonbox",
+        project_desc = "Reference implementation of dragonbox, a float-to-string conversion algorithm based on a beautiful algorithm Schubfach, developed by Raffaello Giulietti in 2017-2018. Dragonbox is further inspired by Grisu and Grisu-Exact.",
+        project_url = "https://github.com/jk-jeon/dragonbox",
+        # NOTE: Update together with v8 and proxy_wasm_cpp_host.
+        version = "6c7c925b571d54486b9ffae8d9d18a822801cbda",
+        strip_prefix = "dragonbox-{version}",
+        sha256 = "2f10448d665355b41f599e869ac78803f82f13b070ce7ef5ae7b5cceb8a178f3",
+        urls = ["https://github.com/jk-jeon/dragonbox/archive/{version}.zip"],
+        use_category = ["dataplane_ext"],
+        extensions = ["envoy.wasm.runtime.v8"],
+        release_date = "2024-10-28",
+        cpe = "N/A",
+    ),
+    fp16 = dict(
+        project_name = "Conversion to/from half-precision floating point formats",
+        project_desc = "Header-only library for conversion to/from half-precision floating point formats.",
+        project_url = "https://github.com/Maratyszcza/FP16",
+        # NOTE: Update together with v8 and proxy_wasm_cpp_host.
+        version = "0a92994d729ff76a58f692d3028ca1b64b145d91",
+        strip_prefix = "FP16-{version}",
+        sha256 = "e66e65515fa09927b348d3d584c68be4215cfe664100d01c9dbc7655a5716d70",
+        urls = ["https://github.com/Maratyszcza/FP16/archive/{version}.zip"],
+        use_category = ["dataplane_ext"],
+        extensions = ["envoy.wasm.runtime.v8"],
+        release_date = "2021-03-20",
+        cpe = "N/A",
+    ),
+    simdutf = dict(
+        project_name = "Unicode validation and transcoding at billions of characters per second",
+        project_desc = "Unicode routines (UTF8, UTF16, UTF32) and Base64: billions of characters per second using SSE2, AVX2, NEON, AVX-512, RISC-V Vector Extension, LoongArch64, POWER. Part of Node.js, WebKit/Safari, Ladybird, Chromium, Cloudflare Workers and Bun.",
+        project_url = "https://github.com/simdutf/simdutf",
+        # NOTE: Update together with v8 and proxy_wasm_cpp_host.
+        version = "7.3.0",
+        sha256 = "512374f8291d3daf102ccd0ad223b1a8318358f7c1295efd4d9a3abbb8e4b6ff",
+        urls = ["https://github.com/simdutf/simdutf/releases/download/v{version}/singleheader.zip"],
+        use_category = ["dataplane_ext"],
+        extensions = ["envoy.wasm.runtime.v8"],
+        release_date = "2025-05-28",
+        cpe = "N/A",
+    ),
+    intel_ittapi = dict(
+        project_name = "Intel ITT API",
+        project_desc = "Intel Instrumentation and Tracing Technology API",
+        project_url = "https://github.com/intel/ittapi",
+        version = "a3911fff01a775023a06af8754f9ec1e5977dd97",
```

Comment:
This is from 2021. Are we sure that we want to use this version?

Reply:
this is the version set up stream